android-components/components/concept/engine/src/main/java/mozilla/components/concept/engine/webextension/WebExtensionDelegate.kt

```diff
@@ -110,6 +110,18 @@ interface WebExtensionDelegate {
         onPermissionsGranted: ((Boolean) -> Unit),
     ) = Unit
 
+    /**
+     * Invoked whenever the installation of a [WebExtension] failed.
+     *
+     * @param extension the extension that failed to be installed, can be null in cases where
+     * the extension couldn't be downloaded or the extension couldn't be parsed.
```

Comment:
```suggestion
     * @param extension the extension that failed to be installed. It can be null when the
     * extension couldn't be downloaded or the extension couldn't be parsed for example.
```